### PR TITLE
feat: add monitoring event

### PR DESCRIPTION
**Pull request by shivanshuit914**

### CODE REVIEW

File: app/event/raise-event.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const { PublishEvent } = require('ffc-ahwr-event-publisher')
+const { eventQueue } = require('../config').mqConfig
+
+const raiseEvent = async (event, status = 'success') => {
+  const eventPublisher = new PublishEvent(eventQueue)
+
+  const eventMessage = {
```
Comment (marcinmo-kainos) [inline app/event/raise-event.js:7]: if it's a monitoring event it'd be best to stick to this name and be very consistent about it.

raise-event.js file please rename to raise-monitoring-event.js

the folder please rename from event to monitoring-event

File: docker-compose.yaml [hunk 2]
```diff
@@ -15,6 +15,7 @@ services:
       APPINSIGHTS_CLOUDROLE: ffc-ahwr-eligibility-local
       APPLICATIONINSIGHTS_CONNECTION_STRING: ${APPLICATIONINSIGHTS_CONNECTION_STRING}
       APPLY_SERVICE_URI: ${APPLY_SERVICE_URI:-http://localhost:3000/apply}
+      EVENT_QUEUE_ADDRESS: ${EVENT_QUEUE_ADDRESS:-ffc-ahwr-event}${MESSAGE_QUEUE_SUFFIX}
```
Comment (marcinmo-kainos) [inline docker-compose.yaml:18]: MONITORING_EVENT_QUEUE_ADDRESS ?

**Review by marcinmo-kainos — DISMISSED** [review]

File: app/event/raise-event.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    name: event.name,
+    properties: {
+      id: event.id,
+      sbi: event.sbi,
+      cph: event.cph,
+      checkpoint: process.env.APPINSIGHTS_CLOUDROLE,
+      status,
+      ip: event.ip,
```
Comment (liamwilson-kainos) [inline app/event/raise-event.js:15]: Just a note to check on whether we can handle the IP address from a compliance/security point of view in relation to the other PR comment -> https://github.com/DEFRA/ffc-ahwr-event/pull/5/files

**Review by govTechSatish — APPROVED** [review]
LGTM

**Pull request merged**